### ISSUE TITLE
PHPUnit 6 Support

### ISSUE THREAD

**Issue by craig-davis** [coduo/php-matcher#96]
Is there currently a plan in place to update the PHPUnit assertions to use the new namespaces in PHPUnit 6?

**Comment by ThePixelDeveloper** [comment]
While we wait, the following is enough to get it working. Place it in your autoload file.

``` php
/**
 * Required because the dependency "coduo/php-matcher" doesn't support PHPUnit 6.*
 */
$phpunitCompat = [
    '\PHPUnit_Framework_TestCase'   => '\PHPUnit\Framework\TestCase',
    '\PHPUnit_Framework_Constraint' => '\PHPUnit\Framework\Constraint\Constraint',
];

foreach ($phpunitCompat as $oldClass => $newClass) {
    if (!class_exists($oldClass) && class_exists($newClass)) {
        class_alias($newClass, $oldClass);
    }
}
```

**Comment by norzechowicz** [comment]
> Is there currently a plan in place to update the PHPUnit assertions to use the new namespaces in PHPUnit 6?

Feel free to send a PR, but because that would be a BC breake it can be released with 3.0.0 tag
